src/ngraph/opsets/opset1_tbl.hpp

```diff
@@ -57,6 +57,7 @@ NGRAPH_OP(Asin, ngraph::op::v0)
 NGRAPH_OP(Atan, ngraph::op::v0)
 NGRAPH_OP(AvgPool, ngraph::op::v1)
 NGRAPH_OP(BatchNormInference, ngraph::op::v0)
+NGRAPH_OP(BatchToSpace, ngraph::op::v1)
```

Comment:
We cannot extend opset1 please create opset2.

Reply:
fixed